SpatialGDK/Source/SpatialGDK/Public/Interop/SpatialReceiver.h

```diff
@@ -194,6 +194,7 @@ class USpatialReceiver : public UObject, public SpatialOSDispatcherInterface
 	FOnEntityRemovedDelegate OnEntityRemovedDelegate;
 
 private:
+
```

Comment:
```suggestion
```

Reply:
Added this to be consistent with the `public` - is there a correct way? Is it no empty line for private but one for public, or both no new line?

Reply:
No new line in both.

Reply:
done